apps/hannk/halide/constants.h

```diff
@@ -0,0 +1,18 @@
+#ifndef HANNK_CONSTANTS_H
+#define HANNK_CONSTANTS_H
+
+namespace hannk {
+
+// After subtracing the zero point, we have 9 bits. We can shift
```

Comment:
might make sense to move `max_rank` here too

Reply:
max_rank isn't used anywhere in the Halide code though?